modules/samples/product/process-tools/src/main/java/org/wso2/bps/samples/processcleanup/DBQuery.java

```diff
@@ -48,12 +52,11 @@
 		Properties prop = new Properties();
 
 		try {
-			String configPath =
-					bpsHome + CleanupConstants.REPOSITORY + File.separator + CleanupConstants.CONF +
-					File.separator + CleanupConstants.CLEANUP_PROPERTIES;
+			String configPath = bpsHome + CleanupConstants.REPOSITORY + File.separator + CleanupConstants.CONF +
+								File.separator + CleanupConstants.CLEANUP_PROPERTIES;
 			prop.load(new FileInputStream(configPath));
 		} catch (IOException e) {
-			e.printStackTrace();
+			log.error("Configuration path error.", e);
 			System.exit(0);
```

Comment:
Beacuse this is a console application which provides for the users to clean/remove the retired processes. Therefore users must have to interact with the console to delete the selected retired processes.